URLQueryItemEncoder/URLQueryItemEncoder.swift

```diff
@@ -63,7 +62,7 @@ extension Array where Element == CodingKey {
     let tailCodingKeyString = keysPath.reduce(into: "", {
       $0 += "[\($1.stringValue)]"
     })
-    
```

Comment:
One more request, could you put spaces back here?

Reply:
fixing now....